doc/user/content/sql/create-cluster-replica.md

```diff
@@ -122,6 +102,30 @@ Action | Outcome
 Increase all replicas' sizes | Ability to maintain more dataflows or more complex dataflows
 Add replicas to a cluster | Greater tolerance to replica failure
 
+### Sizes
+
+Valid `size` options are:
+
+- `3xsmall`
+- `2xsmall`
+- `xsmall`
+- `small`
+- `medium`
+- `large`
+- `xlarge`
+- `2xlarge`
+- `3xlarge`
+- `4xlarge`
+- `5xlarge`
+- `6xlarge`
+
+The [`mz_internal.mz_cluster_replica_sizes`](/sql/system-catalog/mz_internal/#mz_cluster_replica_sizes) table lists the CPU, memory, and disk allocation for each replica size.
+
+{{< warning >}}
+The values in the `mz_internal.mz_cluster_replica_sizes` may change at any time.
+You should not rely on them for any kind of capacity planning.
+{{< /warning >}}
+
```

Comment:
I think we should kill "Disk-attached replicas", "Deployment options", and "Sizes" here, and instead say "The `DISK` and `SIZE` options work as they do for `CREATE CLUSTER`, except they apply to only a single cluster replica."